cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -118,23 +123,29 @@ def finish_flow_cell_temp(self, flow_cell_directory_name: str) -> None:
         """
 
         LOG.info(f"Finish flow cell {flow_cell_directory_name}")
-        self.copy_sample_sheet()
 
-        flow_cell_directory_path: Path = Path(self.demux_api.out_dir, flow_cell_directory_name)
-        bcl_converter: str = get_bcl_converter_name(flow_cell_directory_path)
+        flow_cell_directory: Path = Path(self.demux_api.out_dir, flow_cell_directory_name)
+        flow_cell_run_directory: Path = Path(self.demux_api.run_dir, flow_cell_directory_name)
+
+        bcl_converter: str = get_bcl_converter_name(flow_cell_directory)
 
         parsed_flow_cell: FlowCellDirectoryData = parse_flow_cell_directory_data(
-            flow_cell_directory=flow_cell_directory_path,
+            flow_cell_directory=flow_cell_directory,
             bcl_converter=bcl_converter,
         )
 
+        self.copy_sample_sheet(
```

Comment:
Maybe `copy_sample_sheet` belongs in the utils module? Something to consider

Reply:
I moved the function to utils in this pr now